src/components/Timeline.js

```diff
@@ -2,9 +2,18 @@ import React from 'react';
 import './Timeline.css';
 import TimelineEvent from './TimelineEvent';
 
-const Timeline = () => {
-  // Fill in your code here
-  return;
+
+const Timeline = (props) => {
+
+    const varHoldsData = props.map((user, index) => {
+      return (<TimelineEvent key= {index} name={user.person} status={user.status} timeStamp={user.timeStamp}/>)
+    });
+
+  return (
+    <section className="timeline">
+        {varHoldsData}
+    </section>
```

Comment:
`varHoldsData` I keep having to look at where this is declared to remember what data it holds.